eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -509,6 +534,6 @@ function CheckArtifactShaAgainstTagsList($priorExistingTagList, $releaseSha, $ap
 
   if ($unmatchedTags.Length -gt 0 -and !$continueOnError) {
     Write-Host "Tags already existing with different SHA versions. Exiting."
-    exit(1)
+    #exit(1)
```

Comment:
Why is this commented out?

Reply:
This is for testing purpose. Will add back.

Reply:
Some of the test pipeline failed at this step as the tag has already existed.
I have a sync script running in local, so I can update all eng/common changes in all lang repos. 
This is definitely will add back.